gcloud/_apitools/test_exceptions.py

```diff
@@ -0,0 +1,88 @@
+# pylint: skip-file
+import unittest2
+
+
+class Test_HttpError(unittest2.TestCase):
+
+    def _getTargetClass(self):
+        from gcloud._apitools.exceptions import HttpError
+        return HttpError
+
+    def _makeOne(self, *args, **kw):
+        return self._getTargetClass()(*args, **kw)
+
+    def test_ctor(self):
+        RESPONSE = {'status': '404'}
+        CONTENT = b'CONTENT'
+        URL = 'http://www.example.com'
+        exception = self._makeOne(RESPONSE, CONTENT, URL)
+        self.assertEqual(exception.response, RESPONSE)
+        self.assertEqual(exception.content, CONTENT)
+        self.assertEqual(exception.url, URL)
+        self.assertEqual(exception.status_code, 404)
+        self.assertEqual(
+            str(exception),
+            "HttpError accessing <http://www.example.com>: "
+            "response: <{'status': '404'}>, content <CONTENT>")
+
+    def test_FromResponse(self):
```

Comment:
When do you plan to change method names (`FromResponse` here) to adhere to PEP8?